src/router/index.js

```diff
@@ -248,11 +284,12 @@ const router = new VueRouter({
   routes
 })
 
-const waitForBrokerReady = async (_to, _from, next) => {
+router.beforeEach(async (to, from, next) => {
+  // wait for the broker
   await broker.ready.promise
-  next()
-}
-
-router.beforeEach(waitForBrokerReady)
+  if (!store.state.unlockedAt && to.meta?.protect == true && to.name !== 'Splash') {
```

Comment:
This should handle any route without `meta: { protect }` as `protect = true`. Incase we forget to add protect to a route

Reply:
yes if we forget to add the meta field should be ok `to.meta?.protect == true`